services/repology/repology-repositories.tester.js

```diff
@@ -0,0 +1,32 @@
+'use strict'
+
+const t = (module.exports = require('../tester').createServiceTester())
+const { nonNegativeInteger } = require('../validators')
+const { projectWithTwentyRepositories } = require('./repology-test-helpers')
+
+t.create('Existing project')
+  .get('/starship.json')
+  .expectBadge({
+    label: 'in repositories',
+    message: nonNegativeInteger,
+  })
+
+t.create('Non-existent project')
+  .get('/invalidprojectthatshouldnotexist.json')
+  .expectBadge({
+    label: 'in repositories',
+    message: '0',
+  })
+
+t.create('Project with repositories')
```

Comment:
Last item 😄 We can get rid of this test too (and the mocked svg). the svg parsing logic is well tested elsewhere in the codebase already, and if repology were to make any changes to their svg structure then the other two tests would fail anyway